libzfs.pyx

```diff
@@ -837,12 +839,16 @@ cdef class ZFS(object):
             props = configuration_data['props']
             holds = configuration_data['holds']
             mounted = configuration_data['mounted']
+            min_txg = configuration_data['min_txg']
+            max_txg = configuration_data['max_txg']
             properties = {}
             simple_handle = len(props) == 1 and 'name' in props
             snap_data = {}
 
         IF HAVE_ZFS_ITER_SNAPSHOTS == 6:
-            libzfs.zfs_iter_snapshots(handle, simple_handle, ZFS.__snapshot_details, <void*>snap_list, 0, 0)
+            libzfs.zfs_iter_snapshots(
+                handle, simple_handle, ZFS.__snapshot_details, <void*>snap_list, min_txg, max_txg
+            )
         ELSE:
             libzfs.zfs_iter_snapshots(handle, simple_handle, ZFS.__snapshot_details, <void*>snap_list)
```

Comment:
Is this for some old libzfs version support? Do we need this in SCALE?

Reply:
I believe we don't, will remove this usage.